NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/InvitePlayer.java

```diff
@@ -74,9 +74,12 @@ public boolean execute(CommandSender s, String[] args) {
 			}
 			return true;
 		}
+		if (pType == PlayerType.NOT_BLACKLISTED) {
+			p.sendMessage(ChatColor.RED + "I think we both know that this shouldnt be possible");
```

Comment:
This is a player, who isn't blacklisted, trying to invite themselves into a group? 

Probably a less flippant message would be good, perhaps "You cannot invite yourself into that group".